generators/server/templates/gradle/swagger.gradle.ejs

```diff
@@ -31,7 +31,7 @@ openApiGenerate {
     apiFilesConstrainedTo = [""]
     modelFilesConstrainedTo = [""]
     supportingFilesConstrainedTo = ["ApiUtil.java"]
-    configOptions = [delegatePattern: "true", title: "<%= dasherizedBaseName %>"]
+    configOptions = [delegatePattern: "true", title: "<%= dasherizedBaseName %>"<%_ if (reactive) { _%>, reactive: "true"<%_ } _%>]
```

Comment:
it will add a line return, can we add `<% %>` instead?

Reply:
You mean 

```suggestion
    configOptions = [delegatePattern: "true", title: "<%= dasherizedBaseName %>"<% if (reactive) {  %>, reactive: "true"<% } %>]
```

Reply:
exact @atomfrede